README.md

```diff
@@ -948,6 +948,41 @@ assert out.model.name == "my-model"
 assert out.model.uuid == state_in.model.uuid
 ```
 
+## CloudSpec
+
+You can set CloudSpec information in the state.
+
+```python
+import scenario
+
+state = scenario.State(
+    cloud_spec=scenario.CloudSpec(
+        type="lxd",
+        name="localhost",
+        endpoint="https://127.0.0.1:8443",
+        credential=scenario.state.CloudCredential(
+            auth_type="clientcertificate",
+            attributes={
+                "client-cert": "foo",
+                "client-key": "bar",
+                "server-cert": "baz",
+            },
+        ),
+    ),
+    model=scenario.Model(name="my-vm-model", type="lxd"),
+)
+```
+
+The mandatory arguments of CloudSpec are `type` and `name`.
+
+Access CloudSpec by `Model.get_cloud_spec()`:
```

Comment:
Should this part of the example rather show `MyCharm` with a simple method that calls `get_cloud_spec()`? I'm just thinking this example isn't normally how you'd use it -- the `get_cloud_spec` call would be in the charm itself.